solidjs-typescript/.babelrc

```
@@ -0,0 +1,3 @@
+{
+  "presets": ["solid"]
+}
```

Adding a newline at end of file.

```suggestion
}

```